src/pcapi/core/bookings/repository.py

```diff
@@ -203,6 +203,20 @@ def get_validated_bookings_quantity_for_venue(venue_id: int) -> int:
     )
 
 
+def find_offer_ids_booked_by_beneficiaries(users: List[User]) -> List[Offer]:
```

Comment:
J'ai l'impression que le typage correspond bien à l'implémentation et qu'on renvoie une liste d'offres, pas une liste d'identifiants. Donc : 

```suggestion
def find_offers_booked_by_beneficiaries(users: List[User]) -> List[Offer]:
```